go.mod

```diff
@@ -12,6 +12,7 @@ require (
 )
 
 require (
+	github.com/btcsuite/btcd/btcutil v1.1.3
```

Comment:
I think you can delete this line and run go mod tidy, since we are not using it directly anymore.